src/connector/builder.rs

```diff
@@ -237,6 +242,22 @@ impl ConnectorBuilder<WantsProtocols1> {
         })
     }
 
+    /// Override server name for the TLS stack
+    ///
+    /// By default, for each connection hyper-rustls will extract host portion
+    /// of the destination URL and verify that server certificate contains
+    /// this value.
+    ///
+    /// If this method is called, hyper-rustls will instead use this resolver
+    /// to compute the value used to verify the server certificate.
```

Comment:
For consistency with the `with_server_name` docs it feels worth keeping the explicit mention of the destination URL being ignored.

```suggestion
    /// to compute the value used to verify the server certificate. Domain name included in
    /// the URL will not affect certificate validation.
```

Reply:
This case is a bit difference since the URL's domain name may affect certificate validation depending on how the resolver behaves, right?

Reply:
That's a fair point. Leaving it as is sounds good to me :+1: